db/migrate/20200514214910_rename_bookmarks_controller.rb

```diff
@@ -0,0 +1,31 @@
+class RenameBookmarksController < ActiveRecord::Migration[5.2]
+  def up
+    original_controller = 'foreman_tasks_tasks'
+    original_bookmarks = Bookmark.where(controller: original_controller)
+    original_bookmarks_names = original_bookmarks.pluck(:name)
+    
+    new_controller = 'foreman_tasks/tasks'
+    new_bookmarks = Bookmark.where(controller: new_controller)
+    new_bookmarks.find_each do |new_bookmark|
+      name = new_bookmark.name
+      is_name_taken = original_bookmarks_names.include? name
+
+      if is_name_taken
+        original_bookmark = original_bookmarks.where(name: name).first
+        is_duplicated = original_bookmark.query == new_bookmark.query &&
+                        original_bookmark.owner_id == new_bookmark.owner_id &&
+                        original_bookmark.owner_type == new_bookmark.owner_type &&
+                        original_bookmark.public == new_bookmark.public
+
+        if is_duplicated
+          original_bookmark.destroy     
+        else
+          modified_name = "#{SecureRandom.base64(5)}_#{name}"
```

Comment:
should we add the letter 'Z' before the token to move all of the migrated values to the bottom of the dropdown?
and should it be the same token for all of them? for example all of the migrated values will look like `z_10k3n_value`
![Selection_318](https://user-images.githubusercontent.com/26363699/82137911-ca63bc80-9824-11ea-8787-5af6a844db6a.png)

Reply:
We should probably remove the non-letters from the hash for better readability.

Personally I don't have a strong opinion for the `z_` prefix, I think it would be better to have it near the original one:
```suggestion
          modified_name = "#{name}_#{cleanup(SecureRandom.base64(5))}"
```

Reply:
Agree, I will update.. thanks